### PR TITLE
minor cleanup

### PR DESCRIPTION
**Pull request by swalker2m**
Trivial updates and some header copyright year changes (to avoid mixing these in with a future PR).

### CODE REVIEW

File: modules/math/shared/src/main/scala/gsp/math/Wavelength.scala [hunk 1]
```diff
@@ -11,7 +11,7 @@ import monocle.Prism
 /**
  * Exact wavelengths represented as unsigned integral picometers in the range [0 .. Int.MaxValue]
  * which means the largest representable wavelength is 2.147483647 mm.
- * @param toPicometers This wavelength in integral picometers (10^-12 of a meter).
+ * @param toPicometers This wavelength in integral picometers (10^-12^ of a meter).
```
Comment (tpolecat) [inline modules/math/shared/src/main/scala/gsp/math/Wavelength.scala]: My intent was `10^-12` ... does `^foo^` make a superscript in Scaladoc?

File: modules/math/shared/src/main/scala/gsp/math/Wavelength.scala [hunk 2]
```diff
@@ -79,25 +80,19 @@ object Wavelength {
   }
 
   /**
-   * Creates a Wavelength from an Int in Å.
-   * @param Å in the range (0 to 21474836)
-   * @return a Wavelength provided Å is in range
+   * Returns a `UnitConverter` for angstroms in the range 0 to 21474836.
    */
   def fromAngstroms: UnitConverter =
```
Comment (tpolecat) [inline modules/math/shared/src/main/scala/gsp/math/Wavelength.scala]: While we're here is there any reason for these to not be `val`s?

File: modules/math/shared/src/main/scala/gsp/math/Wavelength.scala (hunk 1 above)
Comment (swalker2m) [reply modules/math/shared/src/main/scala/gsp/math/Wavelength.scala]: Yes.  You could argue though that mostly the documentation is read in the source code and so this change is more confusing. I can put it back.

File: modules/math/shared/src/main/scala/gsp/math/Wavelength.scala (hunk 2 above)
Comment (swalker2m) [reply modules/math/shared/src/main/scala/gsp/math/Wavelength.scala]: Yeah that makes sense.

File: modules/math/shared/src/main/scala/gsp/math/Wavelength.scala (hunk 1 above)
Comment (tpolecat) [reply modules/math/shared/src/main/scala/gsp/math/Wavelength.scala]: Ok. Either way. I tried `10⁻¹²` but it's unreadable in my terminal font.

**Pull request merged**